mqtt_exporter.py

```diff
@@ -455,10 +455,23 @@ def clear(self):
 
     def update(self, label_values, value):
```

Comment:
I think the same usecase exists for other Metric types too.
It would fit far better in https://github.com/fhemberger/mqtt_exporter/blob/master/mqtt_exporter.py#L254-L264 
E.g. it could be another config parameter "json" which could contain a path for json that are not flat, but contain multiple layers. 
e.g. if you have a resonse like:
```
{
    "UptimeSec": 3842454,
    "POWER1": "OFF",
    "Wifi": {
        "AP": 1,
        "Channel": 1,
        "Mode": "11n",
        "RSSI": 94,
        "Signal": -53,
    }
}
```
and you want to monitor the signal, you put something like `Wifi.Signal` in the `json` config to point to the right value.